src/Libraries/Microsoft.Extensions.AI.Evaluation.Console/Microsoft.Extensions.AI.Evaluation.Console.csproj

```
@@ -0,0 +1,66 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>A dotnet tool for managing the evaluation data and generating reports.</Description>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>$(NetCoreTargetFrameworks)</TargetFrameworks>
+    <RootNamespace>Microsoft.Extensions.AI.Evaluation.Console</RootNamespace>
+    <!-- EA0000: Use source generated logging methods for improved performance. -->
+    <NoWarn>$(NoWarn);EA0000</NoWarn>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>aieval</ToolCommandName>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Workstream>AIEval</Workstream>
+    <Stage>preview</Stage>
+    <ForceLatestDotnetVersions>true</ForceLatestDotnetVersions>
+    <EnablePackageValidation>false</EnablePackageValidation>
+    <MinCodeCoverage>88</MinCodeCoverage>
+    <MinMutationScore>0</MinMutationScore>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.CommandLine" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.Extensions.AI.Evaluation\Microsoft.Extensions.AI.Evaluation.csproj" />
+    <ProjectReference Include="..\Microsoft.Extensions.AI.Evaluation.Reporting\CSharp\Microsoft.Extensions.AI.Evaluation.Reporting.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <InternalsVisibleToTest Include="Microsoft.Extensions.AI.Evaluation.Console.Tests" />
```

[nitpick] You could consider this instead:
```suggestion
    <InternalsVisibleToTest Include="$(AssemblyName).Tests" />
```

---

Ah this is not always applicable since we have 6 product projects that are tested by 3 test projects. I'd prefer to leave this for now until we start having more 1-1 mappings.